siac/hostcmd.go

```diff
@@ -358,7 +358,7 @@ func hostannouncecmd(cmd *cobra.Command, args []string) {
 	case 1:
 		err = post("/host/announce", "netaddress="+args[0])
 	default:
-		cmd.Usage()
+		cmd.UsageFunc()
```

Comment:
are you sure you rebuilt the binary? `cmd.UsageFunc()` just returns a function, it doesn't print anything.